From 76b0deb795252d518da0d06b448a4bb924a64040 Mon Sep 17 00:00:00 2001
From: "haotian.qu" <haotian.qu@amlogic.com>
Date: Mon, 24 Jun 2024 20:45:41 +0800
Subject: [PATCH 1/3] hdmipnly: t7 support dtoverlay hdmionly config [2/2]

PD#

Problem:
t7 support dtoverlay hdmionly config.

Solution:
t7 support dtoverlay hdmionly config.

Verify:
local

Signed-off-by: haotian.qu <haotian.qu@amlogic.com>
Change-Id: I747b0a5f9fcc9c0f6c8985914201b23d5b9f3218
---
 board/amlogic/t7_an400_lpddr4x/Kconfig | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/board/amlogic/t7_an400_lpddr4x/Kconfig b/board/amlogic/t7_an400_lpddr4x/Kconfig
index c8a412c315b..87221ad23c8 100644
--- a/board/amlogic/t7_an400_lpddr4x/Kconfig
+++ b/board/amlogic/t7_an400_lpddr4x/Kconfig
@@ -24,6 +24,10 @@ config DTB_BIND_KERNEL
     bool "load dtb from boot part but not flash rsv"
     default n
 
+config AML_DT_OVERLAY
+    bool "use dts overlay"
+    default n if HDMITX_ONLY="true"
+
 config AML_REBOOT
        bool "reboot cmd"
        default y
-- 
2.25.1

